elements/lisk-p2p/test/unit/errors.ts

```diff
@@ -12,6 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+
```

Comment:
maybe prettier setting is different?

Reply:
Can you further elaborate your suggestion?